source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -23,20 +23,79 @@ Real-time monitoring will provide visibility into the user's complete infrastruc
 
 *   Target Release: 4.1
 *   Status: Released
-*   Last updated: Mon 24 Jan 2017
+*   Last updated: Mon 12 Jun 2017
 
 ## Description
 
-[![oVirt Metrics Store Architecture](/images/wiki/MetricsStoreArchitecture.jpg)](images/wiki/MetricsStoreArchitecture.jpg)
+In 4.1, Collectd and Fluentd are already installed on the hosts and engine machines.
+We have an Ansible script that configures them on all relevant hosts.
 
-In 4.1, Collectd will be already installed and configured on the hosts.
-We plan to further integrate it with external tools to collect, aggregate and visualize metrics and logs.
+We plan to continue adding additional logs and metrics as required, add pre-defined dashboards and aletring that will detect and notify the user in case something goes wrong in his environment, based on SLA events, thresholds etc.
 
-In the next version, Collectd will be installed and configured with the engine as well.
+### oVirt Metrics Installation Instructions
 
-We also plan on adding Aletring that will detect and notify the user in case something goes wrong in his environment,
-based on SLA events, thresholds etc.
+**oVirt Metrics Store setup -**
 
-## See Also
+**Note:** Currently it should be installed on a new machine, separate from the engine. 
+It can be installed on a dedicated vm. 
 
-[oVirt Metrics – EFK](https://sradcoblog.wordpress.com/2016/07/19/ovirt-metrics-elk/)
+Please follow one of these options, adding SSO in optional:
+ 
+  * [Metrics Store setup on Top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
+
+  OR
+
+  * [Metrics Store setup on Top of OpenShift without oVirt Engine SSO](https://github.com/ViaQ/Main/blob/master/README-mux.md)
+
+[![oVirt Metrics Data Flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
+
+Once you finished this step, you should have:
+
+  * Kibana - <https://kibana.{hostname}>
+  * Openshit portal - <https://openshift.{hostname}>
+
+
+**oVirt Hypervisors and engine setup -**
+
+On upgrade from 4.1.0, oVirt machines already include Fluentd and Collectd packages.
+Now we need to deploy and configure Collectd and Fluentd to send the data to the central Metrics store:
+
+1. Install / Upgrade and setup oVirt Engine 4.1 latest.
+
+2. Install / Upgrade and activate one or more hosts, 4.1 latest.
+
+3. Copy the CA certificate, created in step 1 on the metrics store machine, to the engine machine.
+
+   Run on the metrics store machine:
+
+        # scp /path/to/ca/certificate.cert root@<fqdn/of/engine/machine>:/etc/ovirt-engine-metrics
+
+4. Copy  /etc/ovirt-engine-metrics/config.yml.example  to config.yml.
+
+   Run on the oVirt engine machine:
+
+        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine/config.yml
+
+5. Update on the engine machine the file /etc/ovirt-engine-metrics/config.yml with the following lines:
```

Comment:
Suggest:
"Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines: